department-of-veterans-affairs/caseflow
# Assign hearings date selector

Resolves #6877 

### Description
This PR updates the Assign Hearings page to load RO's upcoming hearing days along with the number of available slots.

### Testing Plan
1. Go to hearings/schedule/assign
1. Select an RO, and confirm their hearing days are displayed in the left hand column
1. Select Central Office, and confirm their hearing days are displayed in the left hand column
